server/template.go

```diff
@@ -38,6 +39,17 @@ func init() {
 		return gitHubToUsernameMappingCallback(githubUsername)
 	}
 
+	funcMap["removeComments"] = func(body string) string {
+		if len(strings.TrimSpace(body)) == 0 {
+			return ""
+		}
+
+		const RegexForCommentMatch string = `(<!--.*?-->)|(<!--[\S\s]+?-->)`
+		reCommentExpression := regexp.MustCompile(RegexForCommentMatch)
```

Comment:
Can we have this call to `MustCompile` the scope of the file, rather than inside of this function, as we only want to run it once? This way we can assure that any panic will surface upon starting the plugin.

Reply:
Hi thank you @mickmister , i am not sure can i do that ?